### DISCUSSION TITLE
Visualize the mutation report in github

### DISCUSSION THREAD

**Discussion by sailinder** [stryker-mutator/stryker-net#2580] [Q&A] [answered]
Hi All,

We are integrating stryker in our github repository using github runners, which are linux containers.
The problem the I have is how to visualize the output in github. Locally it generates a html report which can be opened in a browser.
In a linux container this is not possible. So, I googled a bit and came across https://github.com/marketplace/actions/mutation-report-action but this expect a specific folder. 
So the question is, is there a way to view the output in github?
And is there no need to have a argument to generated a folder, since now it creates a folder with a timestamp. 

I hope to get some answers :).
Thanks in advance.

Regards,

**Comment by rouke-broersma** [comment, marked as answer]
You can change the output path so it becomes deterministic: https://stryker-mutator.io/docs/stryker-net/configuration/#output-string
You can also change the report file name: https://stryker-mutator.io/docs/stryker-net/configuration/#report-file-name-string

**Reply by sailinder** [reply]
Damn, how did I overlook that. Thank you Rouke.
